source/components/molecules/TabNavigator/TabNavigator.tsx

```diff
@@ -24,7 +24,12 @@ const FlexWrapper = styled.View`
   flex: 1;
 `;
 
-const colors: PrimaryColor[] = ['red', 'blue', 'green'];
+const colorsDictionary = {
```

Comment:
If we use enum here it will prevent us from indexing with a string on line 75.
For the sake of clarity I have updated the type to Record<string, Color>.